### ISSUE TITLE
Previously desync free game now sees reliable desyncs on multiple actions

### ISSUE THREAD

**Issue by Synaps3** [rwmt/Multiplayer#272]
My friend is playing on my server in multiplayer. We have two bases, and we have tried with async time both on and off.

We have no mods, just harmony and multiplayer and the two DLCs.

A few of the things that generate reliable desyncs:
Me completing a caravan trade
Him editing stockpiles
Research completing
[Desync-98.zip](https://github.com/rwmt/Multiplayer/files/7827773/Desync-98.zip)
[Desync-99.zip](https://github.com/rwmt/Multiplayer/files/7827774/Desync-99.zip)
[Desync-100.zip](https://github.com/rwmt/Multiplayer/files/7827775/Desync-100.zip)
[Desync-97.zip](https://github.com/rwmt/Multiplayer/files/7827776/Desync-97.zip)
[Synapse's game 8.zip](https://github.com/rwmt/Multiplayer/files/7827779/Synapse.s.game.8.zip)

Attached is the save file and 4 example desync logs.

**Comment by SpideyLee2** [comment]
I'd like to also mention that I've been having these same desyncs. Though I've only been playing on modded servers recently, these servers don't have mods that affect caravan, stockpile, or research functionality.

**Comment by Synaps3** [comment]
> I'd like to also mention that I've been having these same desyncs. Though I've only been playing on modded servers recently, these servers don't have mods that affect caravan, stockpile, or research functionality.

Hey I just wanted to come back and say we narrowed the issue to having ever enabled the beta async time feature on our server. Any save after that exhibited this crash, even after matching time between the two bases (to the hour) and then converting to singleplayer and re-hosting to disable asynctime. We went back through our saves to the first one before we turned on async time and proceeded to play without issue since then.
